src/intro/hello-world.md

```diff
@@ -73,19 +73,13 @@ edition = "2021"
 crate-type = ["cdylib"]  # Compile this crate to a dynamic C library.
 
 [dependencies]
-godot = { git = "https://github.com/godot-rust/gdext", branch = "master" }
+godot = "0.1"
 ```
```

Comment:
I think we should keep both versions, for users who want a crates release and users who want to follow bleeding-edge development. Maybe mention the crates.io one first, and then introduce the GitHub one.